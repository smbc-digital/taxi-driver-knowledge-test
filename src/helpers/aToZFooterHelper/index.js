const showAToZFooter = (show) =>{
    let footer  = document.getElementById('atoz')
    if(footer !== null) 
    {
        if(show){
            footer.classList.remove('hidden')
        }
        else{
            footer.classList.add('hidden')
        }
    }
}

export default showAToZFooter