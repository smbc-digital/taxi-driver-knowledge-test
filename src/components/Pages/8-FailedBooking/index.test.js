import { React } from '../../../helpers/SetupTest'
import { FailedBooking } from './index'
import renderer from 'react-test-renderer'

describe('FailedBooking', () => {
    describe('snapshot', () => {
		it('should render correctly',() => { 
			// Arrange
			const history = { block: jest.fn() }

			// Act
			const tree = renderer
			.create(<FailedBooking history={history} />)
			.toJSON()
		
			// Assert
			expect(tree).toMatchSnapshot()
		})
	})
})
