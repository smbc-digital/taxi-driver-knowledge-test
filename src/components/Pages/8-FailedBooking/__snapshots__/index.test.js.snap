// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FailedBooking snapshot should render correctly 1`] = `
<div
  className="system-failure"
>
  <h1>
    Sorry, something went wrong
  </h1>
  <div
    className="icon"
  >
    <span
      aria-hidden="true"
      className="fa fa-5x fa-exclamation"
    />
  </div>
  <h2 />
  <p>
    We're sorry, something went wrong and we may have taken the money from your account but not ordered your test. Please ring the contact centre and give them your details to make sure your order is complete.
  </p>
  <p>
    <a
      className="button-primary no-margin-bottom"
      href="https://www.stockport.gov.uk/"
    >
      Go back to the hompage
    </a>
  </p>
</div>
`;
